src/BaseApi.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import { LoadObject } from 'brewskey.js-api';
+
+export const doRequest = async (...fetchArgs: Array<any>): Promise<any> => {
+  // eslint-disable-next-line no-undef
+  const response = await fetch(...fetchArgs);
+
+  const responseJson = await response.json();
+  if (!response.ok) {
+    throw new Error(responseJson.error_description);
+  }
+
+  return responseJson;
+};
+
+// todo add some helpers so we won't have to call emitChanges() in everyplace
+// and actully made some helper to avoid boilerplate in every api method
+class BaseApi {
```

Comment:
`BaseAPI`

Reply:
won't it be inconsistent with `DAOApi` ?

Reply:
Ok, what you have is fine then